client/landing/domains/utils.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { moment } from 'i18n-calypso';
+
+export function getMaintenanceMessageFromError( error, translate ) {
+	const maintenanceEndTime = get( error, [ 'data', 'maintenance_end_time' ], null );
+
+	if ( maintenanceEndTime ) {
+		return translate(
+			"There is an active domain maintenance and we can't serve your request at this moment. Please try again %(when)s.",
```

Comment:
I'd like to suggest an alternative wording for this message. How about the following:

"Our domain management system is currently undergoing maintenance and we can't process your request right now. Please try again..."